runners/spark/src/test/java/org/apache/beam/runners/spark/SparkRunnerRegistrarTest.java

```diff
@@ -37,7 +37,10 @@
   @Test
   public void testOptions() {
     assertEquals(
-        ImmutableList.of(SparkPipelineOptions.class, SparkStructuredStreamingPipelineOptions.class),
+        ImmutableList.of(
```

Comment:
Maybe we don't have to do it in this PR, but I'd be in favor of removing this test. It's kind of pointless to test a constant.